README.md

```diff
@@ -172,6 +173,16 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 
 ---
 
+## Blockchain
+- :hammer_and_wrench:[Blockchain Developer Roadmap](https://roadmap.sh/blockchain)
+- 📁 [Ethereum official Documentation](https://ethereum.org/en/developers/docs/)
+- 📁 [Solidity documentation](https://docs.soliditylang.org/en/v0.8.17/)
+- 📹 [Dapp University youtube](https://www.youtube.com/c/DappUniversity)
+- 💡 [Developers Communities](https://github.com/commclassroom/roadmaps/tree/main/Blockchain/Communities)
+- 💡 [Project Ideas](https://dev.to/edge-and-node/the-complete-guide-to-full-stack-web3-development-4g74)
+   
```

Comment:
use this for folder emoji -
```
:file_folder:
```
for video-camera emoji -

```
:video_camera:
````

for bulb emoji

```
:bulb:
```

Reply:
okay